lib/WeBWorK/ContentGenerator/Instructor/StudentProgress.pm

```diff
@@ -177,14 +175,14 @@ sub displaySets ($c) {
 			$score = wwRound(2, $score);
 
 			my $version_data = {
-				version            => $vNum,
-				score              => $score,
-				total              => $total,
-				date               => $dateOfTest,
-				testtime           => $testTime,
-				timeleft           => $timeLeft,
-				problem_scores     => $problem_scores,
-				incorrect_attempts => ''
+				version                    => $vNum,
+				score                      => $score,
+				total                      => $total,
+				date                       => $dateOfTest,
+				testtime                   => $testTime,
+				timeleft                   => $timeLeft,
+				problem_scores             => $problem_scores,
+				problem_incorrect_attempts => $problem_incorrect_attempts
 			};
```

Comment:
None of the changes to this file should be in this pull request.  This is unrelated to the help file update.  It should be in a separate pull request so that others can review the changes you are making here.

Furthermore, I completely disagree with removing the close date from the page.  Removing it from the title is fine, but removing it from the page entirely is not.

Reply:
Will put this in a new pullrequest.